From 5eaaa03619e6a7c2df559312db19568c9527faa6 Mon Sep 17 00:00:00 2001
From: Xylar Asay-Davis <xylarstorm@gmail.com>
Date: Fri, 29 Oct 2021 13:15:00 -0600
Subject: [PATCH] rename python-igraph to igraph in setup.py

---
 conda_package/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conda_package/setup.py b/conda_package/setup.py
index 3245f1c6..bd3f43d8 100755
--- a/conda_package/setup.py
+++ b/conda_package/setup.py
@@ -17,7 +17,7 @@
     'pyamg',
     'pyevtk',
     'pyproj',
-    'python-igraph',
+    'igraph',
     'scikit-image',
     'scipy',
     'shapely',
