diff --git a/conda_package/mpas_tools/mesh/creation/sort_mesh.py b/conda_package/mpas_tools/mesh/creation/sort_mesh.py
index f55375a5..ba7ad3f0 100644
--- a/conda_package/mpas_tools/mesh/creation/sort_mesh.py
+++ b/conda_package/mpas_tools/mesh/creation/sort_mesh.py
@@ -150,7 +150,7 @@ def cell_del2(mesh):
     return csr_matrix((xvec, (ivec, jvec)))
 
 
-if (__name__ == "__main__"):
+def main():
     parser = argparse.ArgumentParser(
         description=__doc__,
         formatter_class=argparse.RawTextHelpFormatter)
@@ -185,3 +185,7 @@ if (__name__ == "__main__"):
             fptr.write("\n")
 
     mesh.to_netcdf(args.sort_file, format="NETCDF4")
+
+
+if (__name__ == "__main__"):
+    main()
